components/remerge/src/engine.rs

```diff
@@ -398,4 +398,50 @@ mod test {
         assert_eq!(um2["bar"], "test");
         um2.assert_tombstones(vec!["foo", "quux"]);
     }
+
+    #[test]
+    fn test_schema_cant_go_backwards() {
+        const FILENAME: &str = "test_schema_go_backwards.sqlite";
+        let _e: RemergeEngine = RemergeEngine::open(FILENAME, &*SCHEMA).unwrap();
```

Comment:
I wasn't sure how best to test this kind of thing. I need a persistent SQLite database so I can compare "previous" to "current". This creates sqlite files but they seem to pile up in `components/remerge/` and clutter `git status`. I'm not sure where these should go or how to get them there.